app/models/test_passage.rb

```diff
@@ -30,8 +36,26 @@ def success?
     correct_percent >= 85
   end
 
+
+  def left_at
+    self.created_at + self.test.passage_time.minutes
+  end
+
+  def passage_time?
+    true unless self.test.passage_time.zero?
+  end
+
+  def secounds_left
+    Time.at(left_at - Time.current).strftime("%s")
+  end
+
   private
 
+  def time_is_over?
+    true if Time.current >= self.left_at && self.passage_time?
```

Comment:
```suggestion
    Time.current >= self.left_at && self.passage_time?
```
сраванения уже возвращают булево

Reply:
Commit: 60c5315eaf774856bceced2ae907856762ab185a